packages/cli/src/plugins/transforms/transform.interface.js

```diff
@@ -4,8 +4,8 @@ const path = require('path');
 // tranform.interface.js
 module.exports = class TransformInterface {
 
-  constructor(request, extensions, contentType = '') {
-    const { context, config } = request.compilation;
+  constructor(request, compilation, { extensions = [], contentType = '' }) {
+    const { context, config } = compilation;
```

Comment:
should we just passthrough context / compilation?  Otherwise it feels like we're just going to end repeating every property and have to keep updating / adding as `context` and `config` get updated, which seems like it would get annoying over time.